### DISCUSSION TITLE
ScenarioMIP experiment names

### DISCUSSION THREAD

**Discussion by matthew-mizielinski** [WCRP-CMIP/CMIP7-CVs#1] [General]
ScenarioMIP are currently working with the following nomenclature for their experiments:

> High (H)
Medium (M)
Medium-Low (ML)
Low (L)
Very Low (VL)
Low Overshoot (LOS)
>
>Concentration driven variant for models that also ran the emission driven variant
High, Concentration driven (HC)
Medium, Concentration driven (MC) 
Low, Concentration driven (LC)
>
>Extensions branching from the scenario indicated by the first letter(s)
>
>H-ext 
H-ext-OS
M-ext 
ML-ext 
L-ext
VL-ext
LOS-ext

How should this best be translated to the form needed for the CVs

**Comment by matthew-mizielinski** [comment]
A simple extrapolation from CMIP6 names would give

Emissions driven
* `esm-ssp-high`
* `esm-ssp-med`
* `esm-ssp-low`
* `esm-ssp-vlow`
* `esm-ssp-low-os`

Concentration driven;
* `ssp-high`
* `ssp-med`
* `ssp-low`

Extensions
* `esm-ssp-high-ext`
* *`esm-ssp-high-os-ext`*
* `esm-ssp-med-ext`
* `esm-ssp-low-ext`
* `esm-ssp-vlow-ext`
* `esm-ssp-low-os-ext`

Note the addition of the overshoot to the high scenario within the extensions list.

Questions that come to mind;
1. I've assumed that "ssp" is still relevant here; it stood for shared socioeconomic pathway. Is this still the right nomenclature? (noting that in CMIP5 to CMIP6 we changed from rcp to ssp)
2. Can/should we swap the `high`, `med`, `low`, `vlow` terms for something more quantitative? Note that `ssp585` represented SSP5 (fossil fueled development) with an 8.5 W m-2 forcing pathway.
3. Do we need to separate the extensions from the core experiments or can these be combined? In CMIP6 we had options on the end date for an experiment. Removing these extensions would likely require the definition of a high overshoot experiment separately.

@taylor13, let me know your thoughts on the above. I think an initial answer might be useful

@CMIP-IPO, If there is any other information that can be shared on the nature of the different scenarios that might help with naming let me know.

**Reply by matthew-mizielinski** [reply]
Found https://wcrp-cmip.org/wp-content/uploads/2024/04/24-04-15_ScenarioMIP-CMIP7-proposal_final.pdf via the experiment [airtable](https://airtable.com/appVPW6XAZfbOZjYM/shrqq9I4NJThwOT9W/tblkc1lkKEtiYKcho)

**Reply by eleanororourke** [reply]
ScenarioMIP are currently finalising their protocol and should be able to get the version they circulate to their SSC to provide the up to date information.

**Reply by johndunne13** [reply]
My understanding is that the "SSP" terminology will NOT be taken forward in CMIP7 due to severe pushback from some nations in AR6

**Reply by matthew-mizielinski** [reply]
> My understanding is that the "SSP" terminology will NOT be taken forward in CMIP7 due to severe pushback from some nations in AR6

Is there a replacement term/acronym from their discussions that we could use in its place?

**Comment by eleanororourke** [comment]
@claudiatebaldi could you review Matt's suggestions and questions above?

**Comment by taylor13** [comment]
In the past, we could easily distinguish between the CMIP3, CMIP5, and CMIP6 "scenario runs" because they were named differently.  I'm worried that generic "high" "med", and "low" would likely be used in the future for *different* scenarios, which might not be that comparable to the previous phase.

**Reply by johndunne13** [reply]
Agreed, just not sure what ScenarioMIP will advise as of yet.

**Comment by johndunne13** [comment]
I have not heard anything, but it would probably be good for the ipo to reach out to scenarioMIP to let them know that this needs to be on their radar.On Sep 24, 2024, at 12:28 PM, Matthew Mizielinski ***@***.***> wrote:﻿

My understanding is that the "SSP" terminology will NOT be taken forward in CMIP7 due to severe pushback from some nations in AR6

Is there a replacement term/acronym from their discussions that we could use in its place?

**Comment by taylor13** [comment]
How about simply "scenario-high", etc.  And to address my concern above, simply include a 7 (for CMIP7) somewhere:  "scenario7-high" or "scenario-high7" or "scenario-high-7".    We would like not to confuse "7" as identifying the scenario but associated with a CMIP phase.  For that reason, I might choose the last option: "scenario-high-7".

**Reply by matthew-mizielinski** [reply]
I'm happy with ssp -> scenario and if we think we need my preference from these is to use `scenario7` (otherwise this might be interpreted as "high scenario number 7")

Including the number and omitting the explicit extensions (assuming that they are dealt with as options for end year) this would give

Emissions driven
* `esm-scenario7-high`
* `esm-scenario7-med`
* `esm-scenario7-low`
* `esm-scenario7-vlow`
* `esm-scenario7-low-os`

Concentration driven;
* `scenario7-high`
* `scenario7-med`
* `scenario7-low`

Extensions
* `esm-scenario7-high-os`

**Reply by taylor13** [reply]
I'm fine with the 7 attached to scenario: "scenario7".  We can think of it as a shorthand for "scenarioMIP7" (although to my knowledge scenarioMIP hasn't ever numbered its phases).

**Comment by claudiatebaldi** [comment]
Hi. First my apologies but I have never received any notification that this discussion was going on. I need to first say that we are in the process of revising the paper and one of the issues is the naming of the low scenarios, which may change slightly. We need to send the whole thing to the SSC in the next few days and we will make a final decision shortly thereafter. I will forward the proposal to add that "scenario7" to the labeling. John is right that we want to stay away from having ssp anywhere near these scenarios' labeling. And as for the question about having a more quantitative labeling, given the switch to emission driven is anybody's guess what the outcomes will be, so we opted for this qualitative nomenclature. Will be back asap with a reaction about that "scenario7". THanks for your patience and thanks Eleanor for alerting me to this.

**Reply by claudiatebaldi** [reply]
SSC = Scientific Steering Committee of ScenarioMIP

**Comment by claudiatebaldi** [comment]
Also, a couple of details: The **Medium-Low (ML)** scenario is missing from your list 
ScenarioMIP will provide concentration input **for all six scenarios** (the three specified will have high and medium priority for ESMs that ran the emission-driven versions, as a means to compare outcomes vis-a-vis the role of an active carbon cycle in the whole uncertainty range that will come out of this) so the version without "esm" would apply to all six scenarios.

**Comment by claudiatebaldi** [comment]
A few more consideration/suggestion from Detlef van Vuuren:


Completely up to the climate guys, but as esm is to some degree the default, one could have expected esm (earth system model?) to have the default short name, and the concentration-driven scenarios being given the longer names. As also esms can run concentration driven… would the naming not be nicer to just add “cd” (concentration-driven) and “ed” (emission-driven) in front of all scenarios.
Maybe it is me – but why “scenario7” and not “scen7” (just saves a lot of unnecessary typing).

**Comment by taylor13** [comment]
Hi Claudia and all,

I think "scen7" would be fine if that's what's preferred. Anyone else have an opinion?

For consistency across CMIP, the approximately two dozen simulations that were emission-driven in CMIP6 were all prefixed with "esm-" (for example, "esm-piControl" and "esm-ssp585").  For the sake of users who have gotten used to this convention, I think we should continue it.

thanks for thinking about this and providing your input and feedback,
Karl

**Comment by johndunne13** [comment]
The Strategic Ensemble Design Task Team and CMIP Panel both discussed the
somewhat confusing "esm" designation for CO2 emissions forcing
(particularly as ESMs become more comprehensive may run with emissions of
additional forcings such as methane), but in the end decided to retain this
designation in the current phase to avoid further confusion and defer the
renaming until a clear need arose..

**Comment by eleanororourke** [comment]
Just re-triggering this issue so we can get the scenario names finalised and into the data request. Do these experiments names (based on the format from the thread above but reflecting on the finalised protocol) work?

**Emissions driven**
esm-scen7-h
esm-scen7-m
esm-scen7-ml
esm-scen7-l
esm-scen7-vllo
esm-scen7-vlho

**Concentration driven;**
scen7-hc
scen7-mc
scen7-lc

**Extensions**
esm-scen7-h-ext
esm-scen7-h-ext-os
esm-scen7-m-ext
esm-scen7-ml-ext
esm-scen7-l-ext
esm-scen7-vllo-ext
esm-scen7-vlho-ext

**Reply by claudiatebaldi** [reply]
We just received a comment from one of our collaborators/co-authors that questioned the naming choice, but we (troika) decided not to change the names before submitting the paper to GMD because that would have triggered another round of discussion with the whole SSC and we really really need to submit this paper. We are however considering the possibility that together with the revision process we could eventually settle on different names, because the issues raised make sense. Is it conceivable not to have the naming settled for another significant length of time?

**Reply by claudiatebaldi** [reply]
Also, since we are going to provide everything as concentrations too, the list would have to be complete also for the concentration-driven labels, and the corresponding extensions.

**Reply by eleanororourke** [reply]
@matthew-mizielinski do we need to have final names registered in the CVs and available for the v1.2 data request?

**Reply by matthew-mizielinski** [reply]
@claudiatebaldi

For Data Request purposes we would like provisional experiment names to be set for the middle of January at the latest. These will be used for v1.1 of the data request. After that there are opportunities to update the experiment information until the middle of March when final values will be needed to support v1.2 of the data request. We can use the list below, based on @eleanororourke's note above as the starting point.

| Scenario | emissions driven | emissions driven extension | concentration driven | concentration driven extension |
| :--- | :---: | :---: | :---: | :---: | 
| high | `esm-scen7-h` | `esm-scen7-h-ext` | `scen7-hc` | `scen7-hc-ext` |
| high overshoot extensions  |   | `esm-scen7-h-ext-os` |  | `scen7-hc-ext-os` |
| medium | `esm-scen7-m` | `esm-scen7-m-ext` | `scen7-mc` | `scen7-mc-ext` |
| medium-low | `esm-scen7-ml` | `esm-scen7-ml-ext` | `scen7-mlc` | `scen7-mlc-ext` |
| low | `esm-scen7-l` | `esm-scen7-l-ext` | `scen7-lc` | `scen7-lc-ext` |
| very low, low overshoot | `esm-scen7-vllo` | `esm-scen7-vllo-ext` | `scen7-vlloc` | `scen7-vlloc-ext` |
| very low, high overshoot | `esm-scen7-vlho` | `esm-scen7-vlho-ext` | `scen7-vlhoc` | `scen7-vlhoc-ext` |

I'll follow up next week, but I note that this set appears to have an `esm-` prefix for emissions driven experiments and a `c` suffix (after the characters indicating the scenario) denoting concentration driven.

**Reply by claudiatebaldi** [reply]
Sounds and looks good, thank you!

**Reply by chrisdane** [reply]
wouldn't `{c,e}d-scen{mip_era_integer}-<scenario name>[-ext]` or such be more consistent?

**Reply by durack1** [reply]
Just thinking aloud about this, in CMIP6 we had the same experiment name, with optional extensions 2100-2300 for 7 of the 12 (including the CMIP5 rcp repeats) experiments. It feels arbitrary to me to create an additional experiment which just extends the exact same experiment temporally - will be a little painful then to temporally splice these together if you have different experiment names, potentially RIPFs etc

**Reply by taylor13** [reply]
I would agree with @durack1 .  What is the reason for separate naming?  Why not a single name and indicate that the minimum number of years is something, but extensions beyond that are encouraged?

**Reply by claudiatebaldi** [reply]
I think I will bring up the extension naming choices at the meeting we are having next week, because I think that one issue might be the fundamentally different nature of the scenarios up to 2100 (IAM generated) and the extensions (stylized). But that was the case for CMIP6 as well, so... Maybe a more important issue is that we have two separate extensions from the high scenario. How would that work if we do not have separate names for the extensions? I think that would be a problem, no?Just wanted to also alert you that since the table above was created we have a new scenario which keeps following the high til the second half of the century, then curbs emission strongly (we are calling it for now High-Low). The naming of these things has proven a big headache and we are still in the process of getting to a solution that satisfies the many opinions expressed apropos. In next week meeting we need to get to a resolution!

**Reply by durack1** [reply]
@claudiatebaldi, the good news is that we can continue to iterate on names, UNTIL we have final forcing datasets being written - we would like to identify these data by experiment name if we can, simplifying their downstream use. It would be possible to continue to tweak, but ideally not, obviously any uncaught errors will also need to be dealt with.

The timelines for forcing data delivery are starting to coalesce; however, there are ongoing discussions with the ScenarioMIP folks as the moving target continues. To meet the IPCC AR7 deadlines, the hope is to have "final" forcing data for the first two leading scenarios before Xmas (Friday 19 December 2025 has been flagged - see working timeline [here](https://wcrp-cmip.org/wp-content/uploads/2025/09/25-09-09_CMIP7-Forcings-drop-in_SLIDES.pdf#page=18)).

pinging @znichollscr @jkikstra - as they are leading the scenario forcing work

**Reply by znichollscr** [reply]
Thanks @durack1. As a correction, to meet IPCC AR7 deadlines, we need to have final forcing data for e.g. emissions by October 8 (so ozone can run thereafter), which means we need names by then (or we have to do some painful hope jumping, which I would really rather avoid).

**Reply by matthew-mizielinski** [reply]
All. Following a decision by the WIP, we've deregistered the ScenarioMIP experiments, and five experiments from AerChemMIP and HighResMIP which used the same naming convention in #276 and associated pull requests.

The intention is to restore these as soon as we get the final decision on naming.

**Comment by CMIP-IPO** [comment]
Just want to check if ScenarioMIP need to complete the new Experiment registration form, or if @wolfiex has enough information to fill in the CV?

**Reply by claudiatebaldi** [reply]
Sorry I do not know what this is. Just FYI the paper is submitted, but we have not received a full set of review comments so the issue of possible changes in the scenario names is still pending, if it is of any relevance to this discussion.

**Reply by bethdingley** [reply]
Sorry Claudia - there is an experiment registration form to add experiments to the CVs [here](https://github.com/WCRP-CMIP/CMIP7_CVs/issues/new?template=cmipld_experiment.yml). we have sent this to all AR7 Fast Track experiment proposers today. My question was for @matthew-mizielinski / @durack1 / @wolfiex to see if you need to fill this form in.

**Reply by matthew-mizielinski** [reply]
@bethdingley, leave this to me to sort out

**Comment by claudiatebaldi** [comment]
@eleanororourke We finally - FINALLY - have consensus around the names of the 7 scenarios!

They will be

H. High
HL. High to Low 
M Medium
ML Medium to Low
L Low
LN Low to Negative
VL Very Low

Each of the scenario will have its own extension (and only one).

Let me know what you need besides this. I guess the labels will follow your suggestion, just with the new acronyms above and adding a c for the concentration driven experiments even if I find that redundant, given the use of "esm" for the emission driven experiments

**Reply by eleanororourke** [reply]
Brilliant, thank you @claudiatebaldi and the whole ScenarioMIP team!

Just one query - does LN replace VLLO and VL replace VLHO?

The labels will likely follow the previous suggestions with the LN and VL replacements and perhaps @matthew-mizielinski could look at removing the c as it is somewhat redundant?

**Reply by claudiatebaldi** [reply]
Oh I'm sorry, no: LN replaces VLHO and VL replaces VLLO

**Reply by bethdingley** [reply]
Great, thank you for the clarification

**Reply by znichollscr** [reply]
@dompap FYI in case you want to do another round of your scenario forcings. I guess we're still waiting for confirmation of the SSPs to be used for each marker so can't be final until that is also confirmed/publicised so maybe no point until that is done and you can do a final round

**Reply by dompap** [reply]
@znichollscr thanks I did a recalculation to fix a small FuturePop issue that affected a few small territories in the projections (historical was not affected by this). Once the SSP-scenario mapping have been confirmed I can provide the data very quickly.

**Reply by znichollscr** [reply]
Ok super. Let me send an email to help get that confirmation

**Comment by bethdingley** [comment]
@claudiatebaldi that's great news! Can I just check the mapping from the old names to the new names. We previously had VLLO and VLHO listed. Is the mapping below correct?

VLLO is now LN
VLHO is now VL

**Reply by claudiatebaldi** [reply]
Nope, as I said in the reply to Eleanor is the other way around. 

VLLO -> VL
VLHO -> LN

**Reply by claudiatebaldi** [reply]
(maybe you wrote your question before my reply to Eleanor :-))

**Comment by matthew-mizielinski** [comment]
@claudiatebaldi, all, 

We'll restore the ScenarioMIP experiment names in the next few days with a slight modification to omit the `c` suffix.

**Reply by matthew-mizielinski** [reply]
Prepared in Pull Request #297

**Comment by eleanororourke** [comment]
@claudiatebaldi sorry to bother you again but could we just confirm the scenario extension names too? Are these correct?

H-ext
HL-ext
M-ext-OS
M-ext
ML-ext
L-ext
LN-ext
VL-ext

**Reply by znichollscr** [reply]
oops sorry @eleanororourke there was an email thread which then led to this: https://github.com/CMIP-Data-Request/Harmonised-Public-Consultation/issues/108

Claudia already confirmed the names in that issue via email (and they are different to the list you have above)

**Comment by claudiatebaldi** [comment]
We don’t have an M-ext-OS.

From: eleanororourke ***@***.***>
Date: Monday, October 27, 2025 at 3:03 AM
To: WCRP-CMIP/CMIP7-CVs ***@***.***>
Cc: Tebaldi, Claudia ***@***.***>, Mention ***@***.***>
Subject: Re: [WCRP-CMIP/CMIP7-CVs] ScenarioMIP experiment names (Discussion #1)

Check twice before you click! This email originated from outside PNNL.


@claudiatebaldi<https://github.com/claudiatebaldi> sorry to bother you again but could we just confirm the scenario extension names too? Are these correct?

H-ext
HL-ext
M-ext-OS
M-ext
ML-ext
L-ext
LN-ext
VL-ext

**Comment by znichollscr** [comment]
Closing, names are all sorted and registered
